packages/components/style.css

```diff
@@ -5,29 +5,6 @@
 @import 'tailwindcss/components' layer(l-components);
 @import 'tailwindcss/utilities';
 
-.nextra-banner-container._bg-neutral-900._flex {
-  @apply bg-[#48224e] bg-none;
-}
-
-:root {
-  --nextra-navbar-height: 72px;
-}
-
-html {
-  /* font from tailwindcss/base is overwritten by Nextra styles */
-  font-family: var(--font-sans), ui-sans-serif, system-ui, sans-serif, 'Apple Color Emoji',
-    'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
-}
```

Comment:
we need to move it to guild-layout

Reply:
all right

Reply:
update: we established in Slack DMs that we don't need to add it back

1. can't use next/font from the library
2. before we use next/font, we don't know the CSS Custom Property name
3. it's enough to do the following in the website repo

```tsx
<GuildLayout
  htmlProps={{ className: "font-sans" }}
  headProps={{ 
    children: <style>{`:root { --font-sans: ${neueMontreal.style.fontFamily}; }`
  }}
/>